pay-api/src/pay_api/schemas/schemas/refund_shortname.json

```diff
@@ -1,71 +0,0 @@
-{
-  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
No point in having this file, we'll use the dataclasses in the future to generate this